### PR TITLE
P0 wishlist

### PR DESCRIPTION
**Pull request by itsmebipul00**

### CODE REVIEW

File: checkout.css [hunk 1]
```diff
@@ -49,33 +62,18 @@
     display: grid;
     gap: 0.5rem;
```
Comment (zoef-yt) [inline checkout.css:63]: use px here

**Review by zoef-yt — APPROVED** [review]
The code looks good to me. The only suggestion is that mention in the description what this pr is about everything else looks good to merge. Good job.

**Review by dev-prabhat — APPROVED** [review]
Looks good to me

**Review by Muzammil-khan710 — APPROVED** [review]
Good to merge bipul!

File: checkout.css (hunk 1 above)
Comment (itsmebipul00) [reply checkout.css:63]: I am changing it to 1 rem now ✌️

Comment (itsmebipul00) [reply checkout.css:63]: Changing it to 1 rem

**Pull request merged**